### PR TITLE
Remove unnecessary kubeadm .env files

### PR DESCRIPTION
**Pull request by luxas**
@krzyzacy

### CODE REVIEW

**Comment by krzyzacy** [comment]
(there might be a unit-test you need to change, we'll see)

File: jobs/env/pull-kubernetes-e2e-kubeadm-gce.env [hunk 1]
```diff
@@ -1,10 +0,0 @@
-### job-env
-
-
-GINKGO_PARALLEL=y
```
Comment (krzyzacy) [inline jobs/env/pull-kubernetes-e2e-kubeadm-gce.env]: this is not a comment :-)

you can add a `--ginkgo-parallel=30` to config.json instead

Comment (luxas) [reply jobs/env/pull-kubernetes-e2e-kubeadm-gce.env]: The pull job is inactivated and not used.
Let's check later today if it makes sense to enable ginkgo in parallel on Slack...
If it makes sense, I'll send a PR to do that for all kubeadm jobs :)

Comment (krzyzacy) [reply jobs/env/pull-kubernetes-e2e-kubeadm-gce.env]: okay - you just need to remember :-)

**Pull request merged**